packages/api-plugin-carts/src/resolvers/Mutation/setFulfillmentTypeForItems.js

```diff
@@ -0,0 +1,36 @@
+import { decodeCartItemOpaqueId, decodeCartOpaqueId } from "../../xforms/id.js";
+
+/**
+ * @name Mutation/setFulfillmentTypeForItems
+ * @method
+ * @memberof Cart/GraphQL
+ * @summary resolver for the setFulfillmentTypeForItems GraphQL mutation
+ * @param {Object} parentResult - unused
+ * @param {Object} args.input - an object of all mutation arguments that were sent by the client
+ * @param {String} args.input.cartId - The opaque ID of the cart to add the items to.
+ * @param {String} args.input.cartToken - The anonymous access cartToken that was returned from `createCart`.
+ * @param {String} args.input.fulfillmentType - The fulfillment type to be set fo all items.
+ * @param {String} [args.input.items] - An array of cart items to be assigned to the fulfillment type.
+ * @param {String} args.input.clientMutationId - An optional string identifying the mutation call
+ * @param {Object} context - an object containing the per-request state
+ * @returns {Promise<Object>} setFulfillmentTypeForItemsPayload
+ */
+export default async function setFulfillmentTypeForItems(parentResult, { input }, context) {
```

Comment:
It's good to have core business logic covered by unit tests.

This mutation might belong to the fulfillment plugin instead but the domain boundaries here are not so well defined and it's hard to determine. This is an opinion-based comment I felt like sharing.

Reply:
Yes, adding test cases.

As you mentioned this could go in FFT plugin. But considering that this is mutation is specific for the cart alone, I thought of adding it here.